lib/liquid/date.rb

```diff
@@ -0,0 +1,25 @@
+require 'utils'
+module Liquid
+  module Date
+    def date(input, format)
```

Comment:
There is already a [date filter](https://github.com/Shopify/liquid/blob/e9b649b3455c63859f1b865a8684607d6ff5b050/lib/liquid/standardfilters.rb#L332)